src/state/zustand/swapperStore/selectors.ts

```diff
@@ -42,149 +27,92 @@ export const selectReceiveAddress = (state: SwapperState) => state.receiveAddres
 export const selectFees = (state: SwapperState) => state.fees
 export const selectProtocolFees = (state: SwapperState) => state.fees?.protocolFees
 export const selectTrade = (state: SwapperState) => state.trade
-export const selectSwapperName = (state: SwapperState) =>
-  state.activeSwapperWithMetadata?.swapper.name
-export const selectActiveSwapperWithMetadata = (state: SwapperState) =>
-  state.activeSwapperWithMetadata
+export const selectPreferredSwapper = (state: SwapperState) => state.preferredSwapper
+export const selectActiveSwapperWithMetadata = ({
+  availableSwappersWithMetadata,
+  preferredSwapper,
```

Comment:
Prefer keeping `state` vs. destructured args here for referential stability and consistency purposes (I reckon the former doesn't matter for zustand since these are just syntactic sugar of sorts and not primitives a la redux)